app/ports/models/stats.py

```diff
@@ -17,6 +17,7 @@ class Submission(models.Model):
     platform = models.CharField(max_length=20, default='')
     macports_version = models.CharField(max_length=10)
     cxx_stdlib = models.CharField(max_length=20, default='libc++')
```

Comment:
Didn't we say that we wouldn't put any defaults here?

Reply:
Oh, thank you very much. It is from a previous commit and got overlooked.